src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -42,6 +42,12 @@ class DropdownAutoCompleteMenu extends React.Component {
      */
     busy: PropTypes.bool,
 
+    /**
+     * Hide's the input when there are no items. Avoid using this when querying
+     * results in an async fashion.
+     */
+    emptyHidesInput: PropTypes.bool,
```

Comment:
@MeredithAnya Do `hideInputWhenEmpty`, `hiddenOnEmpty`, or `hiddenWhenEmpty` sound better to you? :bike::house:

Reply:
ooo I like `hiddenOnEmpty`